lib/ts/fetch.ts

```diff
@@ -298,6 +298,9 @@ export default class AuthHttpRequest {
                     }
                     logDebugMessage("doRequest: Retrying original request");
                 } else {
+                    if (response.status === AuthHttpRequest.config.invalidClaimStatusCode) {
+                        onInvalidClaimResponse(JSON.parse(response.headers.get("invalid-claim")!));
```

Comment:
- We should extract `JSON.parse(response.headers.get("invalid-claim")!)` into a function. We can call this function `getInvalidClaimsFromResponse`. This should be compatible with fetch and axios response.
- We decided that we will not be sending invalid-claims as a header, and instead will be sending it as a body.